packages/wallets/src/features/cashier/components/WalletCashierHeader/WalletCashierHeader.tsx

```diff
@@ -1,84 +1,162 @@
-import React from 'react';
-import { useHistory, useLocation } from 'react-router-dom';
+import React, { useEffect, useRef } from 'react';
 import classNames from 'classnames';
+import { useHistory, useLocation } from 'react-router-dom';
 import { useActiveWalletAccount } from '@deriv/api';
-import { WalletCardIcon } from '../../../../components/WalletCardIcon';
-import { WalletGradientBackground } from '../../../../components/WalletGradientBackground';
+import { WalletCardIcon, WalletGradientBackground, WalletText } from '../../../../components';
 import { WalletListCardBadge } from '../../../../components/WalletListCardBadge';
 import useDevice from '../../../../hooks/useDevice';
 import CloseIcon from '../../../../public/images/close-icon.svg';
+import IcCashierAdd from '../../../../public/images/ic-cashier-deposit.svg';
+import IcCashierStatement from '../../../../public/images/ic-cashier-statement.svg';
+import IcCashierTransfer from '../../../../public/images/ic-cashier-transfer.svg';
+import IcCashierWithdrawal from '../../../../public/images/ic-cashier-withdrawal.svg';
+import ResetBalance from '../../../../public/images/plus-thin.svg';
 import './WalletCashierHeader.scss';
 
-const realAccountTabs = ['deposit', 'withdraw', 'transfer', 'transactions'];
-const virtualAccountTabs = ['withdraw', 'transfer', 'transactions', 'reset-balance'];
+type TProps = {
+    hideWalletDetails: boolean;
+};
+
+const realAccountTabs = [
+    {
+        icon: <IcCashierAdd />,
+        path: 'deposit',
+        text: 'Deposit',
+    },
+    {
+        icon: <IcCashierWithdrawal />,
+        path: 'withdraw',
+        text: 'Withdraw',
+    },
+    {
+        icon: <IcCashierTransfer />,
+        path: 'transfer',
+        text: 'Transfer',
+    },
+    {
+        icon: <IcCashierStatement />,
+        path: 'transactions',
+        text: 'Transactions',
+    },
+] as const;
+
+const virtualAccountTabs = [
+    {
+        icon: <IcCashierTransfer />,
+        path: 'transfer',
+        text: 'Transfer',
+    },
+    {
+        icon: <IcCashierStatement />,
+        path: 'transactions',
+        text: 'Transactions',
+    },
+    {
+        icon: <ResetBalance />,
+        path: 'reset-balance',
+        text: 'Reset balance',
+    },
+] as const;
```

Comment:
@heorhi-deriv The icons for deposit and withdrawal tabs are different from the ones used in design.
<img width="618" alt="image" src="https://github.com/binary-com/deriv-app/assets/125039206/c03c2585-292c-43f2-a463-d5218c003157">